client/dashboard/sites/overview/plan-card.tsx

```diff
@@ -26,9 +55,33 @@ export default function PlanCard( { site }: { site: Site } ) {
 			heading={ site.plan?.product_name_short || '\u00A0' }
 			description={ getCardDescription( plan, purchase ) }
 			tracksId="plan"
-			variant={ isLoadingPlan || isLoadingPurchase ? 'loading' : undefined }
+			variant={
+				isLoadingPlan || isLoadingPurchase || isLoadingMediaStorage ? 'loading' : undefined
+			}
 			link={ site.plan?.is_free ? undefined : '/v2/me/billing/active-subscriptions' }
-			bottom={ <div /> }
+			bottom={
+				<VStack spacing={ 4 }>
+					<Stat
+						density="high"
+						strapline={ __( 'Storage' ) }
+						metric={ filesize( mediaStorage?.storage_used_bytes ?? 0, { round: 0 } ) }
+						description={ filesize( mediaStorage?.max_storage_bytes ?? 0, { round: 0 } ) }
+						descriptionAlignment="end"
+						progressValue={ progressBarValue }
+						progressColor={ storageWarningColor }
+						progressLabel={ `${ storageUsagePercent }%` }
+					/>
+					<Stat
+						density="high"
+						strapline={ __( 'Bandwidth' ) }
+						metric="7.2 GB"
```

Comment:
Bandwidth still hardcoded to 7.2 GB. What metric even is this?

There's a similar question in Figma. If this metric is going to change in the future maybe hardcoding it for now is still fine. QOBjujZah0UlGDDdLKZhzi-fi-563_33089#1338305814